### PR TITLE
Update README with instructions on setting up the Google Sheet exporter

### PR DESCRIPTION
**Pull request by SeanZoR**

### CODE REVIEW

**Comment by brafdlog** [comment]
Hi @SeanZoR,
Thanks for this :)
Just to clarify, this works for you in the release version out of the box right? You just want to document how to do this for people running caspion from source code?

**Comment by SeanZoR** [comment]
@brafdlog correct, just for running from source code.

File: README.md [hunk 1]
```diff
@@ -97,6 +97,30 @@ YNAB is a budgeting software. If you want to manage your budget there and have y
   - The `account number` could be the credit card number, or the bank account number. To be sure, you can run the app so it exports transactions to a CSV and get the account number from there.
 - **Click on Save to save the configuration**
 
+### Google Sheet integration setup (optional)
```
Comment (brafdlog) [inline README.md:100]:
```suggestion
### Google Sheets integration setup - when building from source code (optional)
```

File: README.md [hunk 2]
```diff
@@ -97,6 +97,30 @@ YNAB is a budgeting software. If you want to manage your budget there and have y
...
+
+Google Sheet can be setup as an exporter, Once done, the transactions results will appear in a dedicated sheet.
+The release version of Caspion uses Release values for that integration. If you wish though to build/debug this, you will need to setup your own test OAuth 2.0 Client on GCP (Google Cloud Platform). Follow these steps:
```
Comment (brafdlog) [inline README.md:103]:
```suggestion
If you set Google Sheets as an exporter the transactions results will appear in a dedicated sheet.
In the release version of Caspion, this integration will work out of the box. 
If you wish though to build/debug this from source, you will need to setup your own test OAuth 2.0 Client on GCP (Google Cloud Platform). Follow these steps:
```

File: README.md (hunk 1 above)
Comment (brafdlog) [inline README.md:100]:
```suggestion
### Google Sheets integration setup - when building from source code (optional)
```

**Review by brafdlog — COMMENTED** [review]
See small changes to clarify that this is only needed when running from source (as opposed to the ynab steps which are required anyway)

**Comment by SeanZoR** [comment]
Thank you @brafdlog , I have applied your suggestions. On my end it's good for merge.

**Pull request merged**